packages/cozy-client/src/models/file.js

```diff
@@ -30,3 +30,27 @@ export const isNote = file => {
     return true
   return false
 }
+
+/**
+ * Normalize an object representing a io.cozy.file object
+ *
+ * Ensure existence of `_id` and `_type` and define a `path`
+ * on files if provided the parent directory.
+ *
+ * @param {object} file - object representing the file
+ * @param {object} parent - object representing the parent directory
+ * @returns {object} full normalized object
```

Comment:
Is it a private or public method ?

Reply:
Today it's used by cozy-sharing, so public.

I've added the corresponding jsdoc